src/components/atoms/Cart.vue

```diff
@@ -1,22 +1,16 @@
 <template>
 	<div>
 		<img :alt="cart.alt" :src="require(`../../assets/icons/${cart.src}`)" class="cart">
-		<p class="cart-status">{{cartStatus.innerText}}</p>
+		<p class="cart-status">{{cart.cartStatus}}</p>
```

Comment:
is your json is compatible with remote one?

Reply:
Only some parts of remote json are compatible, mainly those which don't have any animation.